sdk/src/main/java/com/amplitude/experiment/DefaultExperimentClient.kt

```diff
@@ -107,6 +109,53 @@ internal class DefaultExperimentClient internal constructor(
             UserSessionExposureTracker(it)
         }
 
+    private val isRunningLock = Any()
+    private var isRunning = false
+
+    override fun start(user: ExperimentUser?): Future<ExperimentClient>? {
+        synchronized(isRunningLock) {
+            if (isRunning) {
+                return null
```

Comment:
Didnt notice this before, but lets just return a completed future, rather than null, then change the function signature.